### ISSUE TITLE
Upload new homepage images (Dashboard with colour contrast and Notify without italics)

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#934]
Image of the dashboard on the homepage: red and green (fail/success) are not obvious in the screenshots and on the Dashboard)

Could add symbol ( X / check?)

https://docs.google.com/document/d/18W-nKAH-54wpNy5XSZeEA31my4XRMxpZRVJ3en4ThAA/edit

Colour contrast between red and green may also be an issue elsewhere: https://notification.alpha.canada.ca/_styleguide

Update second image to remove italics from Notify name.

https://www.figma.com/file/qSW5XL4PpjbC8SyZAl2bkS/Unauthenticated-pages?node-id=110%3A0

**Comment by sastels** [comment]
Do we use red anywhere? I know the homepage has the image

<img width="327" alt="image" src="https://user-images.githubusercontent.com/8228248/84696293-47647d80-af1a-11ea-842d-8716a558abaf.png">

But the real dashboard looks like

<img width="745" alt="image" src="https://user-images.githubusercontent.com/8228248/84696235-27cd5500-af1a-11ea-827f-9813d6613729.png">

Do we just need to update that image on the home page?

**Comment by anikbrazeau** [comment]
Interesting, I thought I'd seen it red before? @ioanacontu do you recall seeing red here?

**Comment by ioanacontu** [comment]
Ha! The mysterious Notify. The image we had from the gov.uk homepage was with initially with red. 
![01-dashboard-en](https://user-images.githubusercontent.com/56094538/84700304-ee4c1800-af20-11ea-8514-70ef4b59aef0.png)

Looking at the gov.uk homepage now, it's all green.. 

<img width="565" alt="Screen Shot 2020-06-15 at 15 54 56" src="https://user-images.githubusercontent.com/56094538/84700337-fefc8e00-af20-11ea-8928-d146bdbc2d3e.png">

**Comment by ioanacontu** [comment]
The app itself is also green. Green as a background for failed sent messages.. not sure it's ideal, then why be bothered to use a colour anyway? We'll probably need to review this in the future. As for now, we can just make it green on the homepage image as well.

**Comment by ioanacontu** [comment]
[New product dashboard screenshots.zip](https://github.com/cds-snc/notification-api/files/4820503/New.product.dashboard.screenshots.zip)

I uploaded the svg images that need to be replaced on the homepage.

**Comment by ioanacontu** [comment]
Oh wait, It should be GC Notify and Notification GC, right? I have to adjust!

**Comment by ioanacontu** [comment]
I attached the images that need replacing on the homepage. 

[New product dashboard screenshots.zip](https://github.com/cds-snc/notification-api/files/4820540/New.product.dashboard.screenshots.zip)

**Comment by mcman12** [comment]
Blocked! Bryan to chat with John to get access to AWS

**Comment by willeybryan** [comment]
@sastels should have access now.

**Comment by ioanacontu** [comment]
New images to be used: 
[New product dashboard screenshots-Jun26.zip](https://github.com/cds-snc/notification-api/files/4839116/New.product.dashboard.screenshots-Jun26.zip)

**Comment by ioanacontu** [comment]
French image, please use this one: 
[[image] Dashboard-FR.svg.zip](https://github.com/cds-snc/notification-api/files/4864840/image.Dashboard-FR.svg.zip)
